src/test/java/net/snowflake/client/jdbc/ConnectionIT.java

```diff
@@ -125,7 +125,7 @@ public void testProdConnectivity() throws SQLException {
 
     Properties properties = new Properties();
 
-    properties.put("user", "fakesuer");
+    properties.put("user", "fakeuser");
```

Comment:
Both `fakesuer` and `fakeuser` were used in code. I didn't see any obvious reason to keep this spelling.